_includes/archive_post.html

```diff
@@ -2,6 +2,6 @@
   <a href="{{ root_url }}{{ post.url }}">
     <b>{{post.title}}</b> -
     {% assign author_key = post.author %}
-    {% for key in author_key %}{% assign author = site.authors[key] %}{% if author %}{% if forloop.index > 1 %},{% endif %} <span class="byline author vcard"><a href ="/author/{{key}}">{{ author.name }}<a/></span>{% endif %}{% endfor %}
+    {% for key in author_key %}{% assign author = site.authors[key] %}{% if author %}{% if forloop.index > 1 %},{% endif %} <span class="byline author vcard"><a href ="/author/{{key}}">{{ author.name }}</a></span>{% endif %}{% endfor %}
```

Comment:
Just correcting some malformed HTML here and in a few other spots. There were clues that updated Liquid/Jekyll are stricter than before, but these probably weren't the source of my compilation problems.